### PR TITLE
[ADD] saas_portal: add 'plan' field in tree view for clients on portal

### PR DESCRIPTION
**Pull request by iledarn**

### CODE REVIEW

**Comment by yelizariev** [comment]
- [x] @iledarn  can you add "group by" to tree view too?

File: saas_portal/models/saas_portal.py [hunk 1]
```diff
@@ -325,7 +325,13 @@ def generate_dbname(self, raise_error=True):
 
     @api.multi
     def create_template(self):
-        self._create_template()
+        try:
+            self._create_template()
+        except:
+            raise
+        else:
+            pass
```
Comment (yelizariev) [inline saas_portal/models/saas_portal.py]: it's the same as without try block

File: saas_portal/models/saas_portal.py [hunk 2]
```diff
@@ -325,12 +325,18 @@ def generate_dbname(self, raise_error=True):
 
     @api.multi
     def create_template(self):
+        res = self._create_template()
+        self.template_id.state = res.get('state')
```
Comment (yelizariev) [inline saas_portal/models/saas_portal.py]: add return here. 
```

2016-11-25 07:03:53,001 16971 INFO 01544-407-e65755--base---server werkzeug: 127.0.0.1 - - [25/Nov/2016 07:03:53] "GET /saas_server/new_database HTTP/1.1" 200 -
2016-11-25 07:03:53,024 16969 WARNING 01544-407-e65755--base---portal openerp.service.model: The method create_template of the object saas_portal.plan can not return `None` !
2016-11-25 07:03:53,076 16969 INFO 01544-407-e65755--base---portal werkzeug: 127.0.0.1 - - [25/Nov/2016 07:03:53] "POST /xmlrpc/2/object HTTP/1.1" 200 -
Traceback (most recent call last):
  File "/mnt/odoo-extra/runbot/static/build/01544-407-e65755/saas.py", line 181, in main
    plan_id = rpc_create_plan(args.get('portal_db_name'))
  File "/mnt/odoo-extra/runbot/static/build/01544-407-e65755/saas.py", line 360, in rpc_create_plan
    rpc_execute_kw(auth, 'saas_portal.plan', 'create_template', [[plan_id]])
  File "/mnt/odoo-extra/runbot/static/build/01544-407-e65755/saas.py", line 268, in rpc_execute_kw
    model, method, rpc_args, rpc_kwargs)
  File "/usr/lib/python2.7/xmlrpclib.py", line 1233, in __call__
    return self.__send(self.__name, args)
  File "/usr/lib/python2.7/xmlrpclib.py", line 1591, in __request
    verbose=self.__verbose
  File "/usr/lib/python2.7/xmlrpclib.py", line 1273, in request
    return self.single_request(host, handler, request_body, verbose)
  File "/usr/lib/python2.7/xmlrpclib.py", line 1306, in single_request
    return self.parse_response(response)
  File "/usr/lib/python2.7/xmlrpclib.py", line 1482, in parse_response
    return u.close()
  File "/usr/lib/python2.7/xmlrpclib.py", line 794, in close
    raise Fault(**self._stack[0])
Fault: <Fault 1: 'Traceback (most recent call last):\n  File "/mnt/odoo-extra/runbot/static/build/01544-407-e65755/openerp/service/wsgi_server.py", line 76, in xmlrpc_return\n    response = xmlrpclib.dumps((result,), methodresponse=1, allow_none=False, encoding=None)\n  File "/usr/lib/python2.7/xmlrpclib.py", line 1086, in dumps\n    data = m.dumps(params)\n  File "/usr/lib/python2.7/xmlrpclib.py", line 633, in dumps\n    dump(v, write)\n  File "/usr/lib/python2.7/xmlrpclib.py", line 655, in __dump\n    f(self, value, write)\n  File "/usr/lib/python2.7/xmlrpclib.py", line 659, in dump_nil\n    raise TypeError, "cannot marshal None unless allow_none is enabled"\nTypeError: cannot marshal None unless allow_none is enabled\n'>
)

2016-11-25 07:01:40
saas.py >>> RPC Execute, saas_portal.server, create, [{'local_host': 'localhost', 'name': '01544-407-e65755--base---server', 'client_id': 'a00abf3c-b2dc-11e6-8862-0242ac110004', 'local_port': '2030'}], {}


```

Comment (iledarn) [reply saas_portal/models/saas_portal.py]: @yelizariev, no, please use _create_template instead of create_template for your RPC

Comment (iledarn) [reply saas_portal/models/saas_portal.py]: @yelizariev , create_temlate is now here only for button func

Comment (iledarn) [reply saas_portal/models/saas_portal.py]: @yelizariev , I agree that I should move  this ``￼self.template_id.state = res.get('state')`` into _create_template also

Comment (yelizariev) [reply saas_portal/models/saas_portal.py]: It's forbiden to call method through rpc that starts with underscore

Comment (yelizariev) [reply saas_portal/models/saas_portal.py]: rename your methods instead

Comment (yelizariev) [reply saas_portal/models/saas_portal.py]: e.g. ``create_template_button``

Comment (yelizariev) [reply saas_portal/models/saas_portal.py]: and rename ``_create_temlate`` to ``create_template``

**Comment by yelizariev** [comment]
Error:


```
  File "/mnt/odoo-extra/runbot/static/build/01558-407-6bbe8e/openerp/addons/saas_portal/models/wizard.py", line 76, in do_upgrade_database
    req, req_kwargs = database_record.server_id._request_server(
AttributeError: 'int' object has no attribute 'server_id'
```

**Comment by yelizariev** [comment]
Some errors on runbot

```
  File "/mnt/odoo-extra/runbot/static/build/01581-407-3a9935/openerp/addons/saas_portal/models/__init__.py", line 4, in <module>
    from . import wizard
ImportError: cannot import name wizard
```

**Pull request merged**